CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+- The GDK for Unity now supports Unity 2019.3. You will need to upgrade your Unity project. [#1272](https://github.com/spatialos/gdk-for-unity/pull/1272)
```

Comment:
```suggestion
- The minimum version of Unity supported by the GDK is now 2019.3. You will need to upgrade your Unity project. [#1272](https://github.com/spatialos/gdk-for-unity/pull/1272)
```

Reply:
makes it clearer why they have to upgrade?

Reply:
Ye